src/test/java/com/emc/mongoose/storage/driver/kafka/integration/CreateRecordTest.java

```diff
@@ -11,52 +12,46 @@
 
 public class CreateRecordTest {
 
-  private static KafkaNodeContainer KAFKA_NODE_CONTAINER;
   private KafkaProducer<String, byte[]> kafkaProducer;
   private static final String TOPIC_NAME = "topic";
   private static final String KEY_NAME = "key";
   private static AdminClient adminClient;
-
-  @BeforeClass
-  public static void createContainers() {
-    try {
-      KAFKA_NODE_CONTAINER = new KafkaNodeContainer();
-    } catch (final Exception e) {
-      throw new AssertionError(e);
-    }
-  }
+  private static Timestamp timestamp;
```

Comment:
the sql specific timestamp is not needed, just add the numeric value to the topic name